### ISSUE TITLE
Change default parameters of Entity and Player

### ISSUE THREAD

**Issue by nskins** [nskins/goby#17]
It is important that the default Entity has the basic stats to fight. For this reason; max_hp, hp, attack, and defense must all change to have default values of 1.

In addition, it is not right that the default Player has pre-defined "good starter" stats. Thus, the Player constructor should inherit those stats mentioned above from the Entity. Additionally, the same can be said of its default battle commands. Change that variable to be 'Array.new.' See the Monster constructor for how this can be done.

Finally, in main.rb, change to those stats from the "old" Player constructor to be the same for the 'player' object in our preset game. This change is content-based so it does not seem that the framework is forcing those stats & commands for the end-user's game.

To summarize:
- [ ] Change the default constructor of Entity's test (enitity_spec.rb) to account for the above changes;
- [ ] Change the default constructor of Player's test (player_spec.rb) to account for the above changes;
- [ ] Change the default values of Entity (entity.rb) as necessary;
- [ ] Change the default values of Player (player.rb) as necessary;
- [ ] Edit main.rb as described.
